cloudhub-2/modules/ROOT/pages/ch2-networking-guide.adoc

```diff
@@ -0,0 +1,189 @@
+= CloudHub 2.0 Networking Guide
```

Comment:
Let's avoid using "guide" in the content. 
Maybe "Networking in CloudHub 2.0" or simply "CloudHub 2.0 Networking"